packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -186,7 +190,8 @@ export type BaseInputProps = FormInputLabelProps &
       | 'creditCardCSC'
       | 'creditCardExpiry'
       | 'creditCardExpiryMonth'
-      | 'creditCardExpiryYear';
+      | 'creditCardExpiryYear'
+      | 'generic';
```

Comment:
Do suggest better name if possible. Having `on` is too vague and anyway it doesn't exist on native

Reply:
Should we call it `default`?

Reply:
It can be confused with default value of that prop. Like `default` won't be the default value of autocomplete prop 😅 🙈

Reply:
I think we can call it `on` and just disable that value in react native with platform-specific types.
https://github.com/razorpay/blade/pull/1242#discussion_r1212835771

Web developers will know what `on` means and will use it properly and it won't show on react native at all